beets/autotag/mb.py

```diff
@@ -285,6 +289,27 @@ def album_info(release):
     artist_name, artist_sort_name, artist_credit_name = \
         _flatten_artist_credit(release['artist-credit'])
 
+    ntracks = sum(len(m['track-list']) for m in release['medium-list'])
+
+    # The MusicBrainz API omits 'artist-relation-list' and 'work-relation-list'
+    # when the release has more than 500 tracks. So we use browse_recordings
+    # on chunks of tracks to recover the same information in this case.
+    if ntracks > BROWSE_MAXTRACKS:
+        log.info(u'Album ' + str(release['id']) + u' has too many tracks')
```

Comment:
```suggestion
        log.debug(u'Album {} has too many tracks', release['id'])
```

Reply:
These should all be `debug`-level logging statements (so they are silent by default).